src/main/java/com/cburch/logisim/std/ttl/Ttl7487.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by the Logisim-evolution developers
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+import com.cburch.logisim.data.Value;
+import com.cburch.logisim.instance.InstancePainter;
+import com.cburch.logisim.instance.InstanceState;
+
+/*
+ * Class simulates a TTL7487 IC
+ * It takes a 4 bit input (A1-A4) and performs one of the following oppertations based on inputs B and c to produce outputs Y1-Y4
+ *   B=0, C=0 output compliment of A
+ *   B=0, C=1 output A unchanged
+ *   B=1, C=0 output all High
+ *   B=1, C=1 output all Low
+ * Likely use is before an adder to give control for addition/subtraction along with ++,--,transparent in conjunction with the adder's carry in bit

```

Comment:
Do you mean with "++" and "--" increment and decrement, respectively?

Reply:
I believed it is clearer now